packages/angular_devkit/build_angular/package.json

```diff
@@ -14,6 +14,8 @@
     "@ngtools/webpack": "0.0.0",
     "ajv": "6.10.0",
     "autoprefixer": "9.5.0",
+    "browserslist": "^4.5.2",
```

Comment:
pin direct dependencies please

Reply:
alternatively, consider vendoring them into our repo so we don't add more transitive dependencies

Reply:
I've pinned it now